recipes-qt/packagegroups/packagegroup-qt5-toolchain-target.bb

```diff
@@ -39,9 +46,7 @@ RDEPENDS_${PN} += " \
     qtbase-tools \
     qttranslations-qtbase \
     qttranslations-qthelp \
-    qtcharts-dev \
-    qtcharts-mkspecs \
-    ${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'qtcharts-qmlplugins', '', d)} \
```

Comment:
RDEPENDS_${PN}_remove = "qtcharts-dev qtcharts-mkspecs qtcharts-qmlplugins"
in your layer where you disable widgets doesn't work? I think it should.

Reply:
It only works for qtcharts-dev and qtcharts-mkspecs.
For qtcharts-qmlplugins it does not work. Maybe because it only appears in RDEPENDS, if opengl is in DISTRO_FEATURES and _remove is executed before...

You can read the exact problem in issue #185

Reply:
We have qtbase with widgets PACKAGECONFIG disabled and qtcharts-5.12.3 is built fine here.

And when I've tried bitbake -e packagegroup-qt5-toolchain-target with RDEPENDS_${PN}_remove = "qtcharts-qmlplugins" it worked as expected.